infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -256,8 +256,9 @@
                 }
                 return true;
             });
+        } else {
+            log.info("triggerSequencerBootstrap: a bootstrap task is already in progress.");
```

Comment:
this is predicated on the fact that the function is being called in a single threaded manner. Please improve this.